### DISCUSSION TITLE
ValueError: dictionary update sequence element #0 has length 1; 2 is required : Error in user_proxy.initiate_chat

### DISCUSSION THREAD

**Discussion by abduljaleeledathol** [microsoft/autogen#187] [Q&A]
![image](https://github.com/microsoft/autogen/assets/83627318/599ed834-9b9c-4f1d-b9a1-e9106dd361b1)
Encountering the above error while running user_proxy.initiate_chat as shown below
user_proxy.initiate_chat(
    assistant,
     message="""What date is today? Compare the year-to-date gain for META and TESLA.""",
)

**Comment by sonichi** [comment]
Please follow https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints
If you use `config_list`, make sure it's a list.

**Reply by abduljaleeledathol** [reply]
Fixed the issue. Thanks for the reply. 
Solved by making config_list file as list and mentioning absolute path for config file with .json extension
